documentation/asciidoc/topics/proc_install_command_line.adoc

```diff
@@ -21,6 +21,8 @@ $ {oc_new_project} ${WATCH_NAMESPACE} <2>
 <1> Creates a project into which you install {ispn_operator}.
 <2> Creates a separate project for the {brandname} cluster that {ispn_operator} controls.
 +
+You can create multiple projects if you want the {ispn_operator} to control more than one {brandname} cluster.
```

Comment:
Operator can control more `{brandname}` clusters even in one namespace

Reply:
thanks @Crumby I've reverted the changes as discussed